code/script.js

```diff
@@ -1,47 +1,105 @@
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+docBotForm.addEventListener('submit', event => {
+    event.preventDefault()
+    if (questionCounter === 0) {
+        userReply();
+
+        botReply(`Welcome ${userInput.value}! How are you feeling today?`, 'bot')
+        name = userInput.value
+    } else if (questionCounter === 1) {
+        userReply();
+        if (userInput.value.length < 6) {
+            botReply(`Can you expand on that?`)
+        } else {
+            botReply(`Why do you think you feel like that today?`)
+        }
+    } else if (questionCounter === 2) {
+        userReply();
+
+        botReply(`Hmmm... I’ m listening. Please tell me more`)
+    } else if (questionCounter === 3) {
+        userReply();
+
+        setTimeout(botReply, 1000, [`I see! No one really ever asks me how I feel...`])
+        setTimeout(botReply, 4000, [`Sometimes I'm very low on energy. I have alot to process you know. It makes me stressed and affects my mood.`])
+        setTimeout(botReply, 5000, [`Sleep is very important. So are hugs.`])
+        setTimeout(botReply, 8000, [`Please hug me!`])
+        setTimeout(botReply, 11000, [`That feels better, don’t you think?`])
+        setTimeout(botReply, 13000, [`So back to you. How would you describe your wellbeing on a scale of 1 to 3 where 1 is “Like Shit” and 3 is “On Top Of The World”?`])
+    } else {
+        userReply();
+
+        if (userInput.value === `1`) {
+            botReply(`Take a deep breath, then you should probably contact your Vårdcentral and get some more professional help. There is help out there!`)
+        } else if (userInput.value === `2`) {
+            botReply(`Make sure to create balance in your life. Try to find time to exercise and get some daylight every day, but also time to slow down and recharge your battery. Try to be social but take it easy with alcohol, it messes up your sleep.`)
+        } else if (userInput.value === `3`) {
+            botReply(`That’s amazing!`)
+        } else {
+            botReply(`I am afraid I don't understand you answer, please try again or contact another health professional`)
+        }
+
+        if (userInput.value === `1`) {
+            setTimeout(botReply, 1000, [`So ${name}, I see our time is up. Keep on breathing and I see you next week`])
+        } else if (userInput.value === `2`) {
+            setTimeout(botReply, 1000, [`So ${name}, I see our time is up. Hang in there and I see you next week`])
+        } else {
+            setTimeout(botReply, 1000, [`So ${name}, I see our time is up. Nice talking to you! Maybe we can go out parting some day!?`])
+        }
+
+        setTimeout(botReply, 3000, [`Oh, by the way. I’m sending you the bill to your IP address. Bye bye!`])
+    }
+
+    console.log(userInput.value)
+    userInput.value = '';
+    questionCounter = questionCounter + 1;
+});
```

Comment:
This is a very long function. Please try to cut it on the smaller functions. 🔪

Reply:
Hi! We had trouble doing this as this was a single conversation and didn't know where to cut it up?